tests/integration/test_api.py

```diff
@@ -470,3 +477,45 @@ def test_annotate_insight_anonymous_then_authenticated(client, mocker):
         server_domain=settings.OFF_SERVER_DOMAIN,
         auth=OFFAuthentication(username="a", password="b"),
     )
+
+
+def test_get_images(client):
+    result = client.simulate_get("/api/v1/images?page=1&count=25")
+    assert result.status_code == 200
+
+    image_model_factory1 = ImageModelFactory(barcode="123")
+    image_model_factory2 = ImageModelFactory(barcode="456")
+
+    image_prediction_factory1 = ImagePredictionFactory(image=image_model_factory1.id)
+    image_prediction_factory2 = ImagePredictionFactory(image=image_model_factory2.id)
```

Comment:
For your information, thanks to factory we can also write it:

```suggestion
    image_prediction1 = ImagePredictionFactory(image__barcode="123")
    image_model1 = image_prediction1.image
    image_prediction2 = ImagePredictionFactory(image__barcode="456")
    image_model2 = image_prediction2.image
```

NB: you should not call the variables image_xxx_factory: the factory is the function that return an object (like a real life industrial factory). 
So you should prefer image_prediction1 and image_model1

Reply:
the image__barcode is a "trick" of Factory Boy, taken from django, which says set myobject.image.barcode to this value (note the double "_" between image and barcode)